components/brave_wallet_ui/components/shared/password-input/style.ts

```diff
@@ -105,6 +105,11 @@ export const ToggleVisibilityButton = styled(WalletButton)<Partial<StyleProps>>`
   padding: 0px;
   width: 18px;
   height: 18px;
+`
+
+export const ToggleVisibilityIcon = styled.div<Partial<StyleProps>>`
+  width: 18px;
+  height: 18px;
```

Comment:
As far as I understood webkit-mask-image and outline border do not work together correctly. Implemented as a separate icon div like in many other places like here https://github.com/brave/brave-core/blob/30e495766a8aabe733e284bd4146d8500fdf1048/components/brave_wallet_ui/components/shared/password-input/index.tsx#L51

Reply:
`ToggleVisibilityButton` is not using props anymore, so I guess we can remove the `<Partial<StyleProps>>` thing.